src/libtorch.cc

```diff
@@ -98,6 +102,12 @@ class ModelState : public BackendModel {
   // Flag to indicate whether inference mode is enabled. Defaults to false.
   bool enable_inference_mode_;
 
+  // Flag to enable/disable various JIT components. Defaults to true.
+  // Tensor fuser configuration will be overridden by configuration of nvfuser.
+  bool enable_tensor_fuser_;
+  bool enable_jit_profiling_;
```

Comment:
Do we want enable_jit_profiling_  to default to true? Would it not cause perf impacts?